Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM golang:1.12 as authenticator-client-builder
+FROM goboring/golang:1.12.17b4 as authenticator-client-builder
 MAINTAINER CyberArk Software Ltd.
 
 ENV GOOS=linux
 ENV GOARCH=amd64
-ENV CGO_ENABLED=0
+ENV CGO_ENABLED=1
```

Comment:
@orenbm I'm asking the @cyberark/community-and-integrations-team to take a look at this change and the copying of libraries below to make sure we understand the implications here and that we agree the change is needed

Can you please make sure the PR description / github commit messages are clear about the motivation for the changes? It is very important for changes like this that we document exactly why we're making these choices for the benefit of community members and also for our future selves. Thanks!